addons/website_links/static/tests/tours/website_links.js

```diff
@@ -38,12 +48,13 @@ registry.category("web_tour.tours").add('website_links_tour', {
             extra_trigger: '#o_website_links_recent_links .truncate_text:first():contains("Contact Us")',
             trigger: '#o_website_links_link_tracker_form #generated_tracked_link:contains("/r/")',
             run: function () {
-                window.location.href = $('#generated_tracked_link').text();
+                window.location.href =
+                    document.querySelector("#generated_tracked_link").textContent;
```

Comment:
@odoo/rd-security => Only changed jQuery by native.

Reply:
@pipu-odoo what was before is not relevant, the question is "is the new code safe ?"